### DISCUSSION TITLE
Support importing annotations from non-default PDF readers

### DISCUSSION THREAD

**Discussion by RobertKidder** [aidenlx/obsidian-zotero#527] [Ideas]
zotero的默认阅读器和专业的pdf阅读器还是有差距，所以我一般选择系统默认的阅读器打开pdf。
但是好像也是因为这个原因，zotlit在导入注释到文献笔记时会失败。使用默认的模板时，注释内容完全不显示。
请问作者，能否增加对其他pdf阅读器的支持。

**Comment by aidenlx** [comment]
在 v2 已经正常支持了

v2 beta 现已发布：https://zotlit.aidenlx.site/docs/install-zotlit
迁移指南：https://zotlit.aidenlx.site/docs/how-to/migrate-from-v1
